core-api/src/main/java/com/optimizely/ab/config/parser/ProjectConfigJacksonDeserializer.java

```diff
@@ -17,73 +17,50 @@
 package com.optimizely.ab.config.parser;
 
 import com.fasterxml.jackson.core.JsonParser;
-import com.fasterxml.jackson.core.type.TypeReference;
+import com.fasterxml.jackson.core.ObjectCodec;
 import com.fasterxml.jackson.databind.DeserializationContext;
 import com.fasterxml.jackson.databind.JsonDeserializer;
 import com.fasterxml.jackson.databind.JsonNode;
-import com.fasterxml.jackson.databind.ObjectMapper;
-import com.fasterxml.jackson.databind.module.SimpleModule;
-import com.optimizely.ab.config.Attribute;
-import com.optimizely.ab.config.EventType;
-import com.optimizely.ab.config.Experiment;
-import com.optimizely.ab.config.FeatureFlag;
-import com.optimizely.ab.config.Group;
-import com.optimizely.ab.config.LiveVariable;
-import com.optimizely.ab.config.ProjectConfig;
-import com.optimizely.ab.config.Rollout;
+import com.optimizely.ab.config.*;
 import com.optimizely.ab.config.audience.Audience;
 
 import java.io.IOException;
 import java.util.List;
 
 class ProjectConfigJacksonDeserializer extends JsonDeserializer<ProjectConfig> {
-
     @Override
     public ProjectConfig deserialize(JsonParser parser, DeserializationContext context) throws IOException {
-        ObjectMapper mapper = new ObjectMapper();
-        SimpleModule module = new SimpleModule();
-        module.addDeserializer(Audience.class, new AudienceJacksonDeserializer());
```

Comment:
Nice catch! Looks like we won't be needing these two classes anymore. Do you mind removing them as well?